app/javascript/src/components/Dashboard/Notes/NoteDetail.jsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+
+import { Clock, MenuVertical } from "neetoicons";
+import { Typography, Avatar, Tooltip, Dropdown, Tag } from "neetoui";
+
+import { calculateTimeDifference, dayTimeFormatter } from "utils/date";
+
+const NoteDetail = ({ title, description, created_at: createdAt }) => (
+  <div className="my-2 flex w-full flex-col rounded-sm border border-solid border-gray-300 p-4 shadow">
+    <div className="flex justify-between">
+      <Typography style="h4" weight="semibold">
+        {title}
+      </Typography>
+      <Dropdown buttonStyle="text" icon={MenuVertical}>
+        <div className="p-1.5">
+          <li>Edit</li>
+          <li>Delete</li>
```

Comment:
Go through the Slack thread.

Reply:
When I use `MenuItem.Button` in place of `li` tag, the browser console throws an error and the page doesn't render.

Reply:
Not helpful. If you can attach a video then it will be helpful for me to debug futher.

Reply:
I think the error is because of the neetoUI version. My code is based on v3 and `MenuItem.Button ` is available in v4.

Reply:
When I uninstall the current `@bigbinary/neetoui` package, i.e. the `v3` package, and install the most recent package, the `v4` package, The app stopped working, so right now I'm continuing with the v3 package.

Reply:
@Ishaan-Singhal Raise a separate issue for upgrading to V4.

Refer the following threads while performing the upgrade process:
- https://github.com/bigbinary/neeto-ui/issues/1332#issuecomment-1255164273
- https://github.com/nancy-binani/wheel/pull/8#discussion_r977596571
- https://neetoui.onrender.com/?path=/docs/migration-guide-v3-to-v4--page

Reply:
Also ensure you have restarted your Rails as well as webpack servers once you upgrade the versions.

Reply:
Ok, I will create a new issue separately to handle the migration from v3 to v4 and handle all the changes in that branch.